client/src/details/accountdetails.cpp

```diff
@@ -98,6 +98,23 @@ ItemDataExtractor *AccountDetails::itemDataExtractor() const
     return mDataExtractor;
 }
 
+void AccountDetails::fillAddressFielsMap(QGroupBox *box, QMap<QString, QString> &map)
+{
+    if (box == mUi->billingAddressGroupBox) {
+        map["street"] = KDCRMFields::billingAddressStreet();
```

Comment:
`map.insert(key, value)` is preferred over `map[key] = value`, which has to create an empty value first before assigning to it.

(that's something I learned from the book Effective STL, iirc, or maybe Effective C++)